### ISSUE TITLE
can-dynamic-import tag breaks auto-reload in the modlet

### ISSUE THREAD

**Issue by Psykoral** [donejs/donejs#1039]
__How often can you reproduce it?__ 

- [X] Always
- [ ] Sometimes
- [ ] Rarely
- [ ] Unable
- [ ] I didn’t try

__Description:__
Using `<can-dynamic-import>` to import a modlet effectively disables the build from watching files in the imported modlet for changes. 

__Steps to reproduce:__

In `index.stache` we use `<can-dynamic-import>` to import a myapp/components/footer/ modlet:
```
<can-dynamic-import from="myapp/components/footer/" isResolved:to="*footerTagReady"></can-dynamic-import>
{{#if *footerTagReady}}
  <my-footer app:from="this"></my-footer>
{{/if}}
```

- If changes are made to `index.stache`, the console rebuilds it automatically:
- Start from:

![screenshot 2017-10-19 13 03 33](https://user-images.githubusercontent.com/1676422/31793273-a498fbdc-b4d3-11e7-9e2c-97d3f168b28b.png)

- Make an edit:

![screenshot 2017-10-19 13 03 48](https://user-images.githubusercontent.com/1676422/31793335-d04a3930-b4d3-11e7-8d72-2f628410c776.png)

- Notice the console reloads the index.stache, and the website reflects the change:

```
Reloading myapp@2.0.0#_app/index.stache!done-autorender@1.3.2#no-zone
```

![screenshot 2017-10-19 13 03 53](https://user-images.githubusercontent.com/1676422/31793401-f20d6f42-b4d3-11e7-89b2-a81f6f072cc4.png)

- If changes are made to any file in `myapp/components/footer/`, such as `footer.js` or `footer.stache`, new changes are not rebuilt. There is no new console output and the webpage is not updated.

- Start with the following in `myapp/components/footer/footer.stache`

![screenshot 2017-10-19 13 04 02](https://user-images.githubusercontent.com/1676422/31793535-5844e3a8-b4d4-11e7-9005-eeaa7f02c1e1.png)

- Make a change, notice that the console does not rebuild the footer component:

![screenshot 2017-10-19 13 04 14](https://user-images.githubusercontent.com/1676422/31793925-6fe07350-b4d5-11e7-88cc-31281e057463.png)

- The website is also not rebuilt and no changes are seen:

![screenshot 2017-10-19 13 03 53](https://user-images.githubusercontent.com/1676422/31793653-a953e3c0-b4d4-11e7-8c13-991a44e84891.png)

- The changes are only seen after a manual refresh of the browser window. 

![screenshot 2017-10-19 13 52 35](https://user-images.githubusercontent.com/1676422/31793730-e2bd7ac2-b4d4-11e7-8e1c-38e179551656.png)

- The Console still has not visibly indicated that it has rebuilt the component. 
`Received client connection`

__Expected results:__
Using `donejs develop` in this case, changes to any file in the myapp/components/footer/ directory should be automatically picked up and rebuilt by the console. 

__Actual results:__
Using `donejs develop` in this case, changes to any file in the myapp/components/footer/ directory **are no longer automatically picked up** and rebuilt by the console. 

__Environment:__

| Software                | Version
| ------------------ | -------
| donejs -V               |  1.0.1
| node -v                  |  6.11.3
| npm -v                   |  5.4.2
| Browser                 |  Any
| Operating system |  Windows 10

**Comment by matthewp** [comment]
Thanks, hope to look into this one soon.

**Comment by matthewp** [comment]
@Psykoral are you still encountering issues with can-dynamic-imports and live-reload?

**Comment by Psykoral** [comment]
I am. Have there been any updates to it that I should be aware of that would resolve it? Has anyone else been able to reproduce it?

**Comment by matthewp** [comment]
There haven't been, just making sure before I tackled it.

**Comment by imaustink** [comment]
Closed via https://github.com/stealjs/steal/pull/1296

**Comment by imaustink** [comment]
Hi @Psykoral, this issue has been fixed and we will be releasing a new version tomorrow.

**Comment by Psykoral** [comment]
Fantastic! You guys rule, as usual. Thanks so much!

**Comment by chasenlehara** [comment]
This was released in [steal 1.5.17](https://github.com/stealjs/steal/releases/tag/v1.5.17)
